data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,543 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader xml:lang="eng">
+       <fileDesc>
+           <titleStmt>
+               <title xml:lang="grc">[Ἑρμοῦ τρισμεγίστου] Ποιμάνδρης</title>
+               <author>Hermes Trimegistus</author>
+               <funder>Center for Hellenic Studies</funder>
+               <respStmt>
+                   <resp>Published original versions of the electronic texts</resp>
+                   <orgName ref="https://www.opengreekandlatin.org">Open Greek and Latin</orgName>
+                   <persName role="principal">Gregory Crane</persName>
+                   <persName role="principal">Leonard Muellner</persName>
+                   <persName role="principal">Bruce Robertson</persName> 
+               </respStmt>
+               <respStmt>
+                   <resp>proofreader</resp>
+                   <persName>
+                       <forename>Hanhardt</forename>
+                       <surname>Lia</surname><!--idno type=""></idno-->
+                   </persName>
+               </respStmt>
+               <respStmt>
+                   <resp>CTS conversion</resp>
+                   <orgName>Center for Hellenic Studies</orgName>
+                   <persName>Charlotte Houghton</persName>
+               </respStmt>
+           </titleStmt>
+           <publicationStmt>
+               <publisher>Open Greek and Latin</publisher>
+               <idno type="filename">tlg1286.tlg001.1st1K-grc1.xml</idno>
+               <pubPlace/>
+               <availability>
+                   <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+               </availability>
+               <date when="2020-08-19"/>
+           </publicationStmt>
+           <sourceDesc>
+               <biblStruct>
+                   <monogr><!--Removed author, since there is more than one--><!--Short version of title; no period; lang attribute-->
+                       <title xml:lang="lat">Hermetica Vol One</title>
```

Comment:
Preferred title is 
Hermetica. The Ancient Greek and Latin Writings which Contain Religious or Philosophic Teachings Ascribed to Hermes Trismegistus.

Then the volume is indicated later in this header, not here.